CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## master / unreleased
```

Comment:
I just realized we didn't mention the introduction of the `local` storage in the changelog. I think it's worth to mention it.